.github/workflows/version.yml

```diff
@@ -12,9 +12,9 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - uses: actions/setup-go@v2
+      - uses: actions/setup-go@v4
         with:
-          go-version: 1.18.x
+          go-version: 1.x
```

Comment:
This change and everything above it complies to our GitLab pipelines. 👍

### But the meta question is:

Should we keep doing this?

OK, nothing **should** break with a Go update. The comment fix you've pushed is a fmt-only thing. But I personally would feel more comfortable with manual Go version bumps. Maybe like our customers who don't install a .0 version. Maybe even like my Nextcloud update policy: vX isn't EOL yet = don't upgrade to v{X+1} yet

Reply:
I don't see any downside in doing it like the way we do it (now).

Reply:
> > Should we keep doing this?
> > OK, nothing **should** break with a Go update. The comment fix you've pushed is a fmt-only thing. But I personally would feel more comfortable with manual Go version bumps.
> 
> I mean if there was a reason to stop doing this, that would not only apply to Go versions but everything. But I really don't see a reason for this unless this would regularly cause severe problems.

The latter is a valid argument.

> > Maybe like our customers who don't install a .0 version. Maybe even like my Nextcloud update policy: vX isn't EOL yet = don't upgrade to v{X+1} yet
> 
> Weren't you pushing the use of generics early on? If we did it that way, you'd have waited for them quite a long time. 😅

This is a special case. 😅 Generics are a killer feature.

Reply:
Related:

> There will not be a Go 2 that breaks Go 1 programs.

– https://go.dev/blog/compat